airbyte-integrations/connector-templates/source-python/setup.py.hbs

```diff
@@ -30,15 +30,6 @@ setup(
     author="Airbyte",
     author_email="contact@airbyte.io",
     packages=find_packages(),
-    install_requires=["airbyte-protocol", "base-python"],
-    package_data={"": ["*.json"]},
-    setup_requires=["pytest-runner"],
-    tests_require=["pytest"],
-    extras_require={
-        # Dependencies required by the main package but not integration tests should go in main. Deps required by
-        # integration tests but not the main package go in tests. Deps required by both should go in
-        # install_requires.
-        "main": [],
-        "tests": ["airbyte-python-test", "pytest"],
-    },
+    install_requires=["airbyte-protocol", "base-python", "pytest==6.1.2"],
```

Comment:
Why don't we put everything in `requirements.txt`?

Reply:
I'm also curious how we decide to put something in `setup.py` vs `requirements.txt.

Reply:
Honestly, there's not a huge reason to use `setup.py` today. It's just a little bit more flexible when packaging data dependencies like JSON files, which we do with schema files or `spec.json`. Will follow up in a separate PR.